Menlo-Legacy/lib/Menlo/CLI/Compat.pm

```diff
@@ -2673,7 +2679,8 @@ sub mirror {
             die <<"DIE";
 TLS issue found while fetching $uri:\n
 $reply->{content}\n
-Please verify your certificates or force an HTTP-only request/mirror at your own risk.
+Please verify your certificates or force an HTTP-only request/mirror
+using --insecure option at your own risk.
```

Comment:
You say "at your own risk" a couple of times...

Using the whole tool is and has always been at my own risk/liability. "at your own risk" sounds like legal terms and seems to suggest there might be a time where someone else accepts the legal liability for the use of this tool.

If the aim is to say non-https is more risky than https it's worth saying "with the risks of non-secure http" or "but be aware of the risks of non-https"... Given how long cpan was non-http and non-https I'd say it might even be worth including a discussion of how risky it actually is to fetch tar balls without host verification or a secret communication channel. 

There's a difference between the risks of usint the cpan cdn without https+verification and the risks of a 192 mirror without https+verification.

Reply:
Thank you for the review. Could you please check if the new message properly addresses your concerns? Thank you!